pyproject.toml

```diff
@@ -4,6 +4,7 @@ target-version = "py38"
 [tool.ruff.lint]
 select = [
     "A",            # flake8-builtins
+    "ARG",          # flake8-unused-arguments
```

Comment:
This might cause failures if a function is following an interface or wrapping another function and needs to define the arguments anyway. But it's fine for here I think.